js/components/checkbox.js

```diff
@@ -6,10 +6,6 @@ Fliplet.FormBuilder.field('checkbox', {
       type: Array,
       default: []
     },
-    defaultValue: {
```

Comment:
What about existing widget instance settings using the default value?

Reply:
@squallstar 
The main reason for this change is that I wasn't able to find how we set data into the `data.fields` array. And the only way to add data there was to change `defaultValue` to the `value`. If you could guide me where I can change that we need to send a `defaultValue` for Checkboxes it would be the best fix in my opinion.

Reply:
@YaroslavOvdii can you elaborate a little more on this:

> _"The main reason for this change is that I wasn't able to find how we set data into the data.fields array"_

Reply:
@squallstar at this [line](https://github.com/Fliplet/fliplet-widget-form-builder/blob/b038c61edfbdbd81053a4c727b94dc89fbb37bfc/js/libs/form.js#L264) we take old field value from the data object. As I have noticed I can write data there only if the default value will be attached to `this.value`. Is there any way we could write `this.defaultValue` there?
I wasn't able to find where this object is set.

Reply:
@YaroslavOvdii `data.fields` is the list of fields configured via the interface as saved in the widget instance settings. I am not understanding what the current problem with the checkboxes are in this situation, but I suppose the problem could be that you are assigning the `value` directly so it wouldn't be parsed by any logic in the `created()` method of the checkboxes (e.g. the logic that splits strings into an array). Is this the case?